lib/favicon-color.js

```diff
@@ -0,0 +1,83 @@
+function Favicon(src) {
```

Comment:
Looks good, but you need some header boilerplate:

```js
/* This Source Code Form is subject to the terms of the Mozilla Public
 * License, v. 2.0. If a copy of the MPL was not distributed with this
 * file, You can obtain one at http://mozilla.org/MPL/2.0/. */

const EXPORTED_SYMBOLS = ['Favicon'];
```

Once that's added, feel free to merge!